Druid/DRMessageSend.class.st

```diff
@@ -3,8 +3,7 @@ Class {
 	#superclass : #DRInstruction,
 	#instVars : [
 		'methodNode',
-		'irGenerator',
-		'typeSystem'
+		'inlineGenerator'
```

Comment:
We need to think about how to move this outside the instruction.
I think that it is ok for now, we still can execute an inlining on one specific instruction (this is important for debugging)